angular.json

```diff
@@ -25,9 +25,10 @@
             "aot": false,
             "assets": ["src/favicon.ico", "src/assets"],
             "styles": [
-              "./node_modules/bootstrap/dist/css/bootstrap.min.css",
               "src/styles.scss",
-              "src/assets/scss/main.scss"
+              "src/assets/scss/main.scss",
+							"node_modules/bootstrap/scss/bootstrap.scss",
```

Comment:
This will override our custom CSS. Kindly move the bootstrap.scss above the syles and main.scss

Reply:
Fixed, @bl4ck4ndbr0wn